app/views/transactions/_search.html.slim

```diff
@@ -1,8 +1,32 @@
-= search_form_for @search, url: search_transactions_path, method: :post do |f|
-  = f.label :date
-  div.input-group
-    = f.text_field :date_gteq, class: 'form-control'
-    span class='input-group-addon'
-      | to
-    = f.text_field :date_lteq, class: 'form-control'
-    = f.submit t('.search'), class: 'btn btn-primary input-group-addon'
+= search_form_for @search, url: search_transactions_path, method: :post, class: 'w-100' do |f|
+  .card.my-2
+    .card-header.pt-3
+      .row
+        .col-md-12
+          h6
+            = t('.filter_by_category')
+    .card-body.d-flex.flex-row.justify-content-around.align-items-center.py-2
+      div
+        h6
+          =t('.select_date_range')
+        div.d-flex.flex-column
+          = f.date_field :transactinable_of_BalanceTransaction_type_date_gt
```

Comment:
@sinpai can't we use snake case? `transactinable_of_balance_transaction_type_date_gt`

Reply:
In practice, snake case isn't applicable here. Ransack matcher doesn't work after changes.